lib/screens/departArriveButtonGroup.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { StyleSheet } from 'react-native';
+import { ButtonGroup } from 'react-native-elements';
+
+export default class departArriveButtonGroup extends Component {
+    constructor() {
+        super()
+        this.state = {
+            selectedIndex: 0
+        }
+        this.updateIndex = this.updateIndex.bind(this)
```

Comment:
Do we even need this? Try running without this line and see if it still works? I don't understand `this` that well in js so I can't tell, but seems like this is just getting overwritten by `updateIndex` declaration below?

Reply:
If it doesn't work, try changing `updateIndex` to an arrow function, I think it forces the `this` in the function body to reference the object

Reply:
I took out the bind line and changed updateIndex to arrow function, still works